cfme/tests/services/test_rest_services.py

```diff
@@ -2434,3 +2434,68 @@ def test_embedded_ansible_cat_item_edit_rest(appliance, request, ansible_catalog
     service_template.reload()
     assert service_template.name == new_data["name"]
     assert service_template.description == new_data["description"]
+
+
+@pytest.mark.meta(automates=[1730813])
+@test_requirements.rest
+@pytest.mark.customer_scenario
+@pytest.mark.parametrize("file_name", ["testgear-dialog.yaml"], ids=[''])
+def test_service_refresh_dialog_fields_default_values(
```

Comment:
**Optional**:
Don't you think we need add something in ```ids```?

Reply:
The test name will be like `test_service_refresh_dialog_fields_default_values[infra]`, so I'm okay with not giving it any ID here. Besides, I couldn't think of any good ID name so I kept it empty.